mesonbuild/modules/gnome.py

```diff
@@ -1112,6 +1124,7 @@ def _get_scanner_ldflags(ldflags: T.Iterable[str]) -> T.Iterable[str]:
         KwargInfo('install_dir_gir', (str, bool, NoneType),
                   deprecated_values={False: ('0.61.0', 'Use install_gir to disable installation')},
                   validator=lambda x: 'as boolean can only be false' if x is True else None),
+        KwargInfo('use_glib_to_compile_repositories', (bool), default=False),
```

Comment:
This is not just a long name, it's an unfortunately long name...

Reply:
It is a stand-in, meant to be bikeshed :)

Reply:
Feels like `use_{new,latest}_compiler`, `use_glib_compiler`, or even `use_glib` would suffice, given that it's an opt-in flag for fairly specific functionality. It can always be explained more fully in the documentation.

Plus, it's meant to be temporary, surely?